packages/accounts/wagmi.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "@wagmi/cli";
+import { kebabCase } from "change-case";
+import dotenv from "dotenv";
+import * as configs from "./plugindefs/index.js";
+import { plugingen } from "./scripts/plugingen.js";
+dotenv.config();
+
+const pluginConfigs = Object.values(configs);
+const pluginRegEx: RegExp = /[pP]lugin/g;
+
+export default defineConfig(
+  pluginConfigs.map((config) => ({
```

Comment:
nit: is there a way to check if there's no diff to the generated file? this re-writes every plugin auto-gen file each time the yarn script is run, right?

Reply:
yea it does and we probs can, but we can probs leverage `nx` and `lerna` to do that by adding an `nx` config for `generate`